CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+# Contributing
+
+## Prerequisites
+
+- `git`
+- `python >=3.8`
+- `nodejs >=18,<19`
+
+## Quick Start
+
+The command below cleans out all built artifacts, ensures a fully-configured JS/Python
+environment, and generates everything up to the documentation website and test reports.
+
+```bash
+git clone https://github.com/jupyterlite/pyodide-kernel
+cd pyodide kernel
+npm run quickstart
+```
+
+> The above installs JupyterLab, which provides the `jlpm` command, a pre-packaged
+> version of `yarn`, and is used instead of `npm` from here on out.
+
+### Serve
```

Comment:
It would be nice to keep a small section about the dev workflow like the original readme / cookiecutter does. Basically what sequence of commands should be run and when to expect changes to be effective (for example after re-running `jlpm run build`).

Reply:
> a small section

I've expanded it some, replicating some of the quick start section, with deeper links to invidividual pieces.

As the "front door" of a package for end users (e.g. shows up on PyPI), I think having too much nodejs-related stuff is actually a shortcoming of the existing cookiecutter output, partially as a hangover from the bad old days of source extensions.